tests/conftest.py

```diff
@@ -43,11 +45,44 @@ def mysql_url(config: configparser.ConfigParser):
             "tests on a {} database".format(section, section)
         )
     connection_conf = config[section]
-    user = connection_conf.get("user", "mlwh")
+    user = connection_conf.get("user", "test")
     password = connection_conf.get("password", "")
     ip_address = connection_conf.get("ip_address", "127.0.0.1")
     port = connection_conf.get("port", "3306")
-    schema = connection_conf.get("schema", "mlwh")
+    schema = connection_conf.get("schema", "mlwarehouse")
+
+    return (
+        f"mysql+pymysql://{user}:{password}@"
+        f"{ip_address}:{port}/{schema}?charset=utf8mb4"
+    )
+
+
+def qcdb_mysql_url(config: configparser.ConfigParser):
```

Comment:
Again, it's a copy paste and rename...  you can factor most of this config reading out.